llvm/test/MachineVerifier/test_g_incompatible_range.mir

```diff
@@ -0,0 +1,34 @@
+# RUN: not --crash llc -mtriple=amdgcn-mesa-mesa3d -mcpu=tahiti -run-pass=none %s -filetype=null 2>&1 | FileCheck %s
+--- |
+  define void @mismatched_range_type() {
+    ret void
+  }
+
+  !0 = !{i64 -4294967295, i64 4294967296}
+  !1 = !{i64 -8589934591, i64 8589934592}
```

Comment:
We don't need 2 i64 ranges with different values. Just need one, or test multiple range types for different instruction types